tools/update-base64.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -e
+# Shell script to update base64 in the source tree to a specific version
+
+BASE_DIR="$( pwd )"
+DEPS_DIR="$BASE_DIR/deps"
+BASE64_VERSION=$1
+
+if [ "$#" -le 0 ]; then
+  echo "Error: please provide an base64 version to update to"
+  echo "	e.g. $0 0.4.0"
+  exit 1
+fi
+
+echo "Making temporary workspace"
+
+WORKSPACE=$(mktemp -d 2> /dev/null || mktemp -d -t 'tmp')
+
+cleanup () {
+  EXIT_CODE=$?
+  [ -d "$WORKSPACE" ] && rm -rf "$WORKSPACE"
+  exit $EXIT_CODE
+}
+
+trap cleanup INT TERM EXIT
+
+BASE64_TARBALL="base64-$BASE64_VERSION.tar.gz"
+
+cd "$WORKSPACE"
+
+echo "Fetching base64 source archive"
+curl -sL -o "$BASE64_TARBALL" "https://api.github.com/repos/aklomp/base64/tarball/v$BASE64_VERSION"
+gzip -dc "$BASE64_TARBALL" | tar xf -
+rm "$BASE64_TARBALL"
```

Comment:
nit: do we need a temporary file?
```suggestion
curl -sL "https://api.github.com/repos/aklomp/base64/tarball/v$BASE64_VERSION" | tar xzf -
```

Reply:
Good catch, fixed!